### PR TITLE
wkbotsfeeder: Add 'mute' command that allows silent notifications from a builder

### PR DESCRIPTION
**Pull request by dpino**
I tested it with an unit test, but I don't know actually how receiving command works.

### CODE REVIEW

File: matrixbot/plugins/wkbotsfeeder.py [hunk 1]
```diff
@@ -115,47 +117,80 @@ def dispatch(self, handler=None):
             except Exception as e:
                 self.logger.error("WKBotsFeederPlugin got error in builder %s: %s" % (builder_name,e))
 
-
-    def command(self, sender, room_id, body, handler):
-        self.logger.debug("WKBotsFeederPlugin command")
-        return
+    def command(self, sender, room_id, body, handler = None):
```
Comment (psaavedra) [inline matrixbot/plugins/wkbotsfeeder.py:120]: body will be something like `mybotname: command arg1 arg2 ...`

File: matrixbot/plugins/wkbotsfeeder.py [hunk 2]
```diff
@@ -115,47 +117,80 @@ def dispatch(self, handler=None):
...
+        if value not in ["on", "off"]:
+            usage()
+            return
+        if value == "on":
+            builders[builderName]["mute"] = True
+        else:
+            builders[builderName].pop("mute")
+        self.bot.send_html(room_id, "<p>Room %s was set to mute %s</p>" % (room_id, value))
 
     def help(self, sender, room_id, handler):
```
Comment (psaavedra) [inline matrixbot/plugins/wkbotsfeeder.py:150]: You could make the [`def usage()`](https://github.com/psaavedra/matrix-bot/pull/56/files#diff-9210bba16129d7d645d5fae49dfab47565db1b8999336573faa6a774a05e6d30R130) a class method and reuse it here.

Or just:
``` python
    def help(self, sender, room_id, handler):
        self.logger.debug("WKBotsFeederPlugin")
        message = "blah ..."
        handler(room_id, message)
```

The rationale behind this is to get this command help listed by the `help` command.

**Review by psaavedra — CHANGES_REQUESTED** [review]
Looks good. Just as couple of suggestions regarding the help command.

**Comment by dpino** [comment]
Thanks for the review. I amended the patch to address suggested changes.

File: matrixbot/plugins/wkbotsfeeder.py [hunk 3]
```diff
@@ -115,47 +117,83 @@ def dispatch(self, handler=None):
...
+        if len(command) == 0:
+            self.help(sender, room_id, handler)
+            return
+        builders = self.settings["builders"]
+        builderName = command[0]
+        if builderName not in builders.keys():
+            self.help(sender, room_id, handler)
+            return
+        value = (command[1] if command[1:] else "on").lower()
+        if value not in ["on", "off"]:
```
Comment (psaavedra) [inline matrixbot/plugins/wkbotsfeeder.py:141]: I like a lot this way to interact with the plugin object. Nice idea.

File: matrixbot/plugins/wkbotsfeeder.py [hunk 4]
```diff
@@ -115,47 +117,83 @@ def dispatch(self, handler=None):
...
+        else:
+            builders[builderName].pop("mute")
+        self.bot.send_html(room_id, "<p>Room %s was set to mute %s</p>" % (room_id, value))
 
     def help(self, sender, room_id, handler):
+        if not handler:
+            return
         self.logger.debug("WKBotsFeederPlugin help")
-        return
+        message = "<p>mute <builder-name> [ON|OFF]<p>"
```
Comment (psaavedra) [inline matrixbot/plugins/wkbotsfeeder.py:154]: Did you tried this? I  think <p> are not needed. Other plugins don't use that.

Comment (psaavedra) [reply matrixbot/plugins/wkbotsfeeder.py:154]: ... for example the plugin/trac.py uses: `message = "%(name)s create This is the issue summary\n" % self.settings`

**Review by psaavedra — CHANGES_REQUESTED** [review]

**Pull request merged**